Source/Analytics/Events.Admin/HealthRisks/HealthRiskModified.cs

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+using System;
+using Dolittle.Artifacts;
+using Dolittle.Events;
+
+namespace Events.Admin.HealthRisks
+{
+    [Artifact("ab9f13e6-f68b-4683-96fe-13434fd34516", 1)]
+    public class HealthRiskModified : IEvent 
+    {
+        public Guid Id { get; }
```

Comment:
Should this be a HealthRiskId and not a Guid?

Reply:
Not for Events no, we should use primitive types - like the strings and ints below - in events